torchaudio/compliance/kaldi.py

```diff
@@ -731,11 +735,15 @@ def _get_LR_indices_and_weights(orig_freq, new_freq, output_samples_in_unit, win
     return min_input_index, weights
 
 
+@torch.jit.export
 def _lcm(a, b):
+    # type: (int, int) -> int
     return abs(a * b) // fractions.gcd(a, b)
 
 
+@torch.jit.export
```

Comment:
By the way, compilation is now recursive, and so functions called within functions that are compiled no longer need to be annotated.